plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js

```diff
@@ -178,7 +178,7 @@ const Blueprint = () => {
 					<h4>{ __( 'Import', 'woocommerce' ) }</h4>
 					<p>
 						{ __(
-							'Import .json file, max size 50 MB. Only one Blueprint can be imported at a time.',
+							'Import a .json file. You can only import one Blueprint at a time.',
```

Comment:
[nitpick] You can import only one Blueprint at a time - the position of only highlights the limitation in quantity